packages/runtime/src/createGatewayRuntime.ts

```diff
@@ -671,11 +696,8 @@ export function createGatewayRuntime<
       }
 
       unifiedGraphFetcher = () =>
-        handleUnifiedGraphConfig(
-          // @ts-expect-error TODO: what's up with type narrowing
-          config.supergraph,
-          configContext,
-        );
+        // @ts-expect-error TODO: what's up with type narrowing
+        handleUnifiedGraphConfig(config.supergraph, configContext);
```

Comment:
This comment was intentionally set above config.supergraph argument originally because then it's allowing an error only on the argument (compared to now, allowing an error on the function and both arguments).

```suggestion
        handleUnifiedGraphConfig(
          // @ts-expect-error TODO: what's up with type narrowing
          config.supergraph,
          configContext,
        );